themes/finna2/templates/organisationinfo/elements/location-selection.phtml

```diff
@@ -0,0 +1,80 @@
+<!-- START of: finna - organisationinfo/elements/location-selection.phtml -->
+<?php if (!$orgInfo): ?>
+  <?=$this->transEsc('organisation_info_no_schedule'); ?>
+  <?php return; ?>
+<?php endif; ?>
+<div class="navi-section map-ui" id="consortium-service-point-section">
+  <div class="locations-title">
+    <h2><?=$this->transEsc('organisation_info_navi_service')?></h2>
+  </div>
+  <div class="location-filters">
+    <?php if (count($orgInfo['list'] ?? []) > 1): ?>
+      <div class="location-filter">
+        <form class="form-inline searchbox-office map-ui">
+          <div class="input-group">
+            <label for="location_select" class="hidden"><?=$this->transEscAttr('Search')?></label>
+            <select id="location_select" class="js-location-select">
+                <option value=""><?=$this->transEsc('organisation_info_autocomplete', ['%%count%%' => count($orgInfo['list'])])?></option>
+              <?php foreach ($orgInfo['list'] as $item): ?>
+                <?php
+                  $name = $item['name'];
+                  if ($city = $item['address']['city'] ?? null) {
+                    $name .= ", $city";
+                  }
+                ?>
+                <option value="<?=$this->escapeHtmlAttr($item['id'])?>"><?=$this->escapeHtml($name)?></option>
+              <?php endforeach; ?>
+            </select>
+            <div class="js-location-unavailable hidden"><?=$this->transEsc('organisation_info_location_not_available')?></div>
+          </div>
+        </form>
+      </div>
+    <?php endif; ?>
+  </div>
+  <?php if ($notification = $orgInfo['consortium']['finna']['notification'] ?? ''): ?>
+    <div>
+      <p class="consortium-notification"><?=$this->cleanHtml($notification)?></p>
+    </div>
+  <?php endif; ?>
+
+  <div class="location-information">
+    <div class="js-location-loader hidden">
+      <?=$this->icon('spinner') ?> <?=$this->transEsc('loading_ellipsis')?>
```

Comment:
Tähän` <span aria-live="polite"><?=$this->transEsc('loading_ellipsis')?></span>`

Reply:
Lisätty.